handlers/reverseProxy/reverseProxy.go

```diff
@@ -0,0 +1,33 @@
+package reverseProxy
```

Comment:
Should the directory name and the package name be `reverseproxy/reverseproxy.go`?